modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp

```diff
@@ -236,7 +236,7 @@ long assetClassPK = DLAssetHelperUtil.getAssetClassPK(fileEntry, fileVersion);
 				<div class="sidebar-section">
 
 					<%
-					boolean isLatestVersion = fileVersion.equals(fileEntry.getLatestFileVersion());
```

Comment:
@slnn You should be able to use dlViewFileEntryDisplayContext.getFileVersion() here, inside this method, there is a cache.

Reply:
OK, got it